server/src/main/java/org/candlepin/resource/OwnerContentResource.java

```
@@ -289,6 +298,12 @@ public ContentData updateContent(@PathParam("owner_key") String ownerKey,
         }
 
         existing = this.contentManager.updateContent(content, owner, true);
+        try {
+            ownerManager.refreshOwnerForContentAccess(owner);
+        }
+        catch (Exception e) {
+            log.error("Unable to handle content access certs.", e.getStackTrace().toString());
```

This should re-throw, eating the exception will cause confusion.